GOVERNANCE.md

```diff
@@ -24,12 +35,14 @@
 * Subproject Maintainer: a maintainer can also review and merge pull requests.
 * TAP Editor: maintainers of the specification and TAPs.
 * Consensus Builder: the project's founder and consensus builder, is able to
-  veto any issues put to the Steering Committee vote.
+  veto any issues put to the Steering Committee vote.  If the consensus builder
+  leaves the project permanently, the steering committee elects another consensus
+  builder.
 
 Contributors who have not actively contributed to the project, through PRs and
 Issues, or communicated on project forums (GitHub, Slack, Mailing list) for
-longer than 12 months may be involuntarily moved down the contributor ladder,
-or asked to move to an emeritus status.
+longer than 2 months may be involuntarily moved down the contributor ladder,
```

Comment:
2 months is a fairly short amount of time, I think we should give folks at least a 3 month/12 week grace period.